bench/wireshark/optimized/packet-pgm.c.ll

```diff
@@ -1116,12 +1116,17 @@ paritystr.exit.i:                                 ; preds = %385, %384, %358
   %456 = trunc nuw nsw i64 %455 to i32
   %457 = load ptr, ptr %24, align 8
   %458 = call noalias ptr @wmem_alloc(ptr noundef %457, i64 noundef 8192) #6
-  %.not497.i = icmp eq i32 %456, 0
-  br i1 %.not497.i, label %._crit_edge.thread.i, label %.lr.ph.i.outer
+  %.not497.i = icmp ult i8 %451, 4
+  br i1 %.not497.i, label %._crit_edge.thread.i, label %.lr.ph.preheader.i
 
-.lr.ph.i.outer:                                   ; preds = %438, %.thread
-  %indvars.iv.i.ph = phi i64 [ %indvars.iv.next.i204, %.thread ], [ 0, %438 ]
-  %.not463.i = phi i1 [ true, %.thread ], [ false, %438 ]
+.lr.ph.preheader.i:                               ; preds = %438
+  %umax.i = call i32 @llvm.umax.i32(i32 %456, i32 1)
+  %wide.trip.count.i = zext nneg i32 %umax.i to i64
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597406008